### DISCUSSION TITLE
WLED over network

### DISCUSSION THREAD

**Discussion by mdb17** [awawa-dev/HyperHDR#765] [Q&A] [answered]
I tried searching for the answer but didn’t find it specifically so I figured I would just ask.

Is it possible to setup HyperHDR up so that it controls a strip that is connected to a WLED controller over WiFi?

the strip I want to control is remote from the source so I am considering using one of these boards running WLED.
https://quinled.info/quinled-dig-uno/

Would that be possible ?

**Comment by awawa-dev** [comment, marked as answer]
From the beginning WLED over wifi is supported by two drivers: udpraw and dedicated WLED driver. Latest v20beta is using WLED Audio-Reactive-Led-Strip protocol to handle it.
